site/zenodo_rdm/migrator/transform/entries/users.py

```diff
@@ -57,9 +57,11 @@ def _displayname(self, entry):
 
     def _profile(self, entry):
         """Returns the profile."""
-        return {
-            "full_name": entry.get("full_name"),
-        }
+        res = {}
```

Comment:
I don't think this is necessary, but if we don't have a "full name", the profile should be an empty dict by default.